test/pageobjects/register.ts

```diff
@@ -18,6 +18,44 @@ class Registration extends Page {
   get masterPasswordInput() {
     return $("#register-form_input_master-password");
   }
+
+  async masterPasswordBitError() {
+    let masterPasswordInput = await $("#register-form_input_master-password");
```

Comment:
You already have it defined on the line 18, please re-use

Reply:
corrected